smart-contracts/scripts/upgrades/2021-11-blocklist/runbook.md

```diff
@@ -6,11 +6,19 @@ If you are trying to upgrade the BridgeBank contract so that it knows how to use
 
 ```
 MAINNET_URL=https://eth-mainnet.alchemyapi.io/v2/your-alchemy-id
-MAINNET_PRIVATE_KEYS=XXXXXXXX,YYYYYYYYYY
+MAINNET_PRIVATE_KEYS=XXXXXXXX,YYYYYYYY,ZZZZZZZZ
```

Comment:
I think it's more descriptive to have 3 separate value, and MAINNET_PRIVATE_KEYS concats them.